kivy/uix/video.py

```diff
@@ -130,13 +171,21 @@ class Video(Image):
     _video_load_event = None
 
     def __init__(self, **kwargs):
+        # B/C case source is given. use it as video source
+        if 'source' in kwargs:
+            kwargs['video_source'] = kwargs['source']
+        # Case preview is given
+        if 'preview_source' in kwargs:
+            kwargs['source'] = kwargs['preview_source']
+
         self._video = None
         super(Video, self).__init__(**kwargs)
-        self.fbind('source', self._trigger_video_load)
+        self.fbind('video_source', self._trigger_video_load)
+        self.fbind('source', self._bc_video_load)
```

Comment:
```suggestion
        self.fbind('preview_source', self._bc_video_load)
        if self.preview_source:
            self.source = self.preview_source
```
We only need to update `source` when `preview_source` changes. We shouldn't update `video_source` from whatever the user set it to. But we need to make sure it's updated here in case it was changed in co-inherited class in `super`.